code/game/jobs/job/civilian.dm

```diff
@@ -130,7 +130,7 @@
 	name = "Quartermaster"
 	jobtype = /datum/job/qm
 
-	uniform = /obj/item/clothing/under/rank/cargo
+	uniform = /obj/item/clothing/under/rank/cargo/qm
```

Comment:
Why?

Reply:
There were duplicate definitions of /obj/item/clothing/under/rank/cargo—I didn't know what to think as to whether it was deliberate or not.
On one hand, the quartermaster came in with the same uniform indicated on the cargo technician's uniform. I can understand if that may have been the intention.
On the other hand, there were two definitions for that uniform. I should also point out that historically, /obj/item/clothing/under/rank/cargo was used for the quartermaster and cargotech for cargo technicians, but the latter cargotech was seemingly deleted, which I can still see how it may have been a minor bit of human error. Additionally, Amory made an icon dedicated to the quartermaster uniform that wasn't used anywhere